diff --git a/CMakeLists.txt b/CMakeLists.txt
index 68b553d..2dfaa2e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -156,7 +156,6 @@ endif ()
 add_definitions (-DOIIO_INTERNAL=1)
 
 list (APPEND CMAKE_MODULE_PATH
-      "${PROJECT_SOURCE_DIR}/src/cmake/modules"
       "${PROJECT_SOURCE_DIR}/src/cmake")
 
 include (GNUInstallDirs)
@@ -235,7 +234,6 @@ if (OIIO_BUILD_TOOLS AND NOT BUILD_OIIOUTIL_ONLY)
     add_subdirectory (src/iinfo)
     add_subdirectory (src/maketx)
     add_subdirectory (src/oiiotool)
-    add_subdirectory (src/testtex)
     add_subdirectory (src/iv)
 endif ()
 
diff --git a/src/dicom.imageio/CMakeLists.txt b/src/dicom.imageio/CMakeLists.txt
index e8a2fbb..06252e0 100644
--- a/src/dicom.imageio/CMakeLists.txt
+++ b/src/dicom.imageio/CMakeLists.txt
@@ -4,7 +4,7 @@
 
 if (DCMTK_FOUND)
     add_oiio_plugin (dicominput.cpp
-                     LINK_LIBRARIES DCMTK::DCMTK
+                     LINK_LIBRARIES DCMTK::ofstd DCMTK::dcmdata DCMTK::dcmimage
                      DEFINITIONS "-DUSE_DCMTK=1")
 else ()
     message (WARNING "DICOM plugin will not be built, no DCMTK")
diff --git a/src/include/CMakeLists.txt b/src/include/CMakeLists.txt
index 18b7632..ce54870 100644
--- a/src/include/CMakeLists.txt
+++ b/src/include/CMakeLists.txt
@@ -64,7 +64,7 @@ install (FILES ${detail_headers}
          DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/${PROJECT_NAME}/detail
          COMPONENT developer)
 
-if (INTERNALIZE_FMT OR OIIO_USING_FMT_LOCAL)
+if (0)  # Conan: Don't try to copy fmt libraries to target install folder
     set (fmt_headers_base_names)
     foreach (header_name core.h format-inl.h format.h ostream.h printf.h
              std.h base.h chrono.h)
diff --git a/src/libOpenImageIO/CMakeLists.txt b/src/libOpenImageIO/CMakeLists.txt
index 49e76e2..063087e 100644
--- a/src/libOpenImageIO/CMakeLists.txt
+++ b/src/libOpenImageIO/CMakeLists.txt
@@ -166,6 +166,7 @@ target_link_libraries (OpenImageIO
             ${BZIP2_LIBRARIES}
             ZLIB::ZLIB
             $<TARGET_NAME_IF_EXISTS:Boost::thread>
+            tsl::robin_map
             ${CMAKE_DL_LIBS}
         )
 
