diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3ff009229..a9ac90a4a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -154,7 +154,6 @@ endif ()
 add_definitions (-DOIIO_INTERNAL=1)
 
 list (APPEND CMAKE_MODULE_PATH
-      "${PROJECT_SOURCE_DIR}/src/cmake/modules"
       "${PROJECT_SOURCE_DIR}/src/cmake")
 
 include (GNUInstallDirs)
@@ -233,7 +232,6 @@ if (OIIO_BUILD_TOOLS AND NOT BUILD_OIIOUTIL_ONLY)
     add_subdirectory (src/iinfo)
     add_subdirectory (src/maketx)
     add_subdirectory (src/oiiotool)
-    add_subdirectory (src/testtex)
     add_subdirectory (src/iv)
 endif ()
 
diff --git a/src/cmake/compiler.cmake b/src/cmake/compiler.cmake
index 7f680b892..429861b00 100644
--- a/src/cmake/compiler.cmake
+++ b/src/cmake/compiler.cmake
@@ -26,8 +26,8 @@ message (VERBOSE "CMAKE_CXX_COMPILE_FEATURES = ${CMAKE_CXX_COMPILE_FEATURES}")
 ###########################################################################
 # C++ language standard
 #
-set (CMAKE_CXX_STANDARD 14 CACHE STRING
-     "C++ standard to build with (14, 17, 20, etc.)")
+# set (CMAKE_CXX_STANDARD 14 CACHE STRING
+#      "C++ standard to build with (14, 17, 20, etc.)")
 set (DOWNSTREAM_CXX_STANDARD 14 CACHE STRING
      "C++ minimum standard to impose on downstream clients")
 set (CMAKE_CXX_STANDARD_REQUIRED ON)
diff --git a/src/cmake/externalpackages.cmake b/src/cmake/externalpackages.cmake
index fe581e723..f0b8823d8 100644
--- a/src/cmake/externalpackages.cmake
+++ b/src/cmake/externalpackages.cmake
@@ -160,7 +160,7 @@ else ()
 endif()
 
 # From pythonutils.cmake
-find_python()
+# find_python()
 if (USE_PYTHON)
     checked_find_package (pybind11 REQUIRED VERSION_MIN 2.4.2)
 endif ()
diff --git a/src/include/CMakeLists.txt b/src/include/CMakeLists.txt
index 18b7632a3..ce54870dd 100644
--- a/src/include/CMakeLists.txt
+++ b/src/include/CMakeLists.txt
@@ -64,7 +64,7 @@ install (FILES ${detail_headers}
          DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/${PROJECT_NAME}/detail
          COMPONENT developer)
 
-if (INTERNALIZE_FMT OR OIIO_USING_FMT_LOCAL)
+if (0)  # Conan: Don't try to copy fmt libraries to target install folder
     set (fmt_headers_base_names)
     foreach (header_name core.h format-inl.h format.h ostream.h printf.h
              std.h base.h chrono.h)
diff --git a/src/libOpenImageIO/CMakeLists.txt b/src/libOpenImageIO/CMakeLists.txt
index aeb7b7f93..b25c01b7f 100644
--- a/src/libOpenImageIO/CMakeLists.txt
+++ b/src/libOpenImageIO/CMakeLists.txt
@@ -164,6 +164,7 @@ target_link_libraries (OpenImageIO
             ${BZIP2_LIBRARIES}
             ZLIB::ZLIB
             $<TARGET_NAME_IF_EXISTS:Boost::thread>
+            tsl::robin_map
             ${CMAKE_DL_LIBS}
         )
 
