diff --git a/CMakeLists.txt b/CMakeLists.txt
index e16167a..be60055 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -47,8 +47,7 @@ ELSE()
   SET(CMAKE_C_STANDARD 99)
 ENDIF()
 
-SET(CMAKE_CXX_STANDARD 11)
-SET(CMAKE_CXX_STANDARD_REQUIRED ON)
+FIND_PACKAGE(mariadb-connector-c REQUIRED)
 
 SET(PRODUCT_NAME "MariaDB C++ Connector")
 SET(PRODUCT_MANUFACTURER "MariaDB")
@@ -565,7 +564,7 @@ IF(MARIADB_LINK_DYNAMIC)
                                              ShLwApi
                 DOC "Path to ShLwApi.Lib. Usually found in $ENV{programfiles\(x86\)}\\Windows Kits\\10\\Lib")
     ELSE()
-      FIND_LIBRARY(CCLIB libmariadb.so)
+      SET(CCLIB mariadb-connector-c::mariadb-connector-c)
       IF (${CCLIB} STREQUAL "CCLIB-NOTFOUND")
         # It won't be found by linker either unless user does some magic before build, so we actually could stop here with error
         MESSAGE(STATUS "C/C library has not been found")
@@ -580,6 +579,7 @@ IF(MARIADB_LINK_DYNAMIC)
                       /usr/local/include/mariadb
                       /usr/local/include/mysql)
                 #PATH_SUFFIXES mariadb mysql)
+      FIND_FILE(CCHEADER NAMES "mysql.h")
       IF (${CCHEADER} STREQUAL "CCHEADER-NOTFOUND")
         # Again should probably stop
         MESSAGE(STATUS "C/C headers have not been found")
@@ -692,7 +692,7 @@ ADD_DEPENDENCIES(${LIBRARY_NAME} DEPENDENCIES_FOR_PACKAGE)
 ########## Packaging ##########
 
 # MSI
-IF(WIN32)
+IF(False)
   IF(WITH_MSI)
     ADD_CUSTOM_COMMAND(TARGET ${LIBRARY_NAME} POST_BUILD
       COMMAND ${CMAKE_COMMAND} ARGS -DDRIVER_LIB_DIR=$<TARGET_FILE_DIR:${LIBRARY_NAME}>
